tests/clipping_area/to_polyfile_test.py

```diff
@@ -3,8 +3,8 @@
 
 from django.contrib.gis.geos import MultiPolygon, Polygon, MultiLineString, LineString
 
+import osmaxx.utils.polyfile_helpers
 from osmaxx.clipping_area.to_polyfile import create_poly_file_string
-from osmaxx.clipping_area import utils
```

Comment:
Why not
```python
from osmaxx.utils.polyfile_helpers import parse_poly_string
```
? When using `parse_poly_string` below, we don't need the additional context of where it's from to tell what its name means.